apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/behandling/Behandling.kt

```diff
@@ -37,12 +37,18 @@ data class ForenkletVedtak(
 )
 
 data class Utbetalingsinfo(
-    val beloep: Int,
-    val kontonummer: String,
-    val virkningsdato: LocalDate,
-    val grunnbeloep: Grunnbeloep,
+    val grunnbeloep: Int,
+    val soeskenjustering: Boolean,
+    val beregningsperioder: List<Beregningsperiode>
+)
+
+data class Beregningsperiode(
+    val datoFOM: LocalDate,
+    val datoTOM: LocalDate?,
+    val grunnbeloep: Int,
     val antallBarn: Int,
-    val soeskenjustering: Boolean
+    val utbetaltBeloep: Int,
+    val trygdetid: Int
```

Comment:
Blir trygdetid brukt til noe i brev, eller er det for fremtidig utvidelse?

Reply:
Fremtidig utvidelse. Kunne kanskje vært en separat PR, men det får så være nå. 🤷